packages/simplebar/src/simplebar.js

```diff
@@ -448,7 +448,9 @@ export default class SimpleBar {
         ? this.contentEl.scrollHeight
         : this.contentEl.scrollHeight - this.minScrollbarWidth) >
       Math.ceil(this.axis.y.track.rect.height);
-
+    */
+   this.axis.x.isOverflowing = this.contentEl.scrollWidth > this.contentEl.clientWidth;
+   this.axis.y.isOverflowing = this.contentEl.scrollHeight > this.contentEl.clientHeight; // Set isOverflowing to false if user explicitely set hidden overflow
```

Comment:
Here you totally ignored some of the check that were in this logic previously: for example the code was testing for `this.scrollbarWidth` and you remove that so I suppose this won't work in some cases (for example if user has floating scrollbar, like on mac OSX).